source/renderer/app/config/analyticsConfig.ts

```diff
@@ -1,25 +1,28 @@
 import { Network } from '../../../common/types/environment.types';
 
-export const ANALYTICS_API_ENDPOINT =
-  'https://daedalusqa2.matomo.cloud/matomo.php';
+export const ANALYTICS_API_ENDPOINT = 'https://matomo.cw.iog.io/matomo.php';
 export const PRIVACY_POLICY_LINK =
   'https://static.iohk.io/terms/iog-privacy-policy.pdf';
-export const DEV_MODE_SITE_MAP_ID = 1;
+
+// ID used when Daedalus is launched from nix-shell
+export const DEV_MODE_SITE_MAP_ID = 11;
```

Comment:
Is it possible to define the ID as a string? The magic numbers are ugly IMO

Reply:
`MatomoTracker` expects a number as site ID, so I would have to cast those strings to a number later on, which introduces room for error (NaN). Could we stick to number? :)

Reply:
Ok, that's sad but lets do it then 👍